addons/website_slides/wizard/slide_channel_invite_views.xml

```diff
@@ -34,20 +35,16 @@
                             </group>
                             <field name="can_edit_body" invisible="1"/>
                             <field name="body" class="oe-bordered-editor" widget="html_mail" invisible="not send_email" readonly="not can_edit_body" force_save="1"/>
-                            <group invisible="not send_email">
-                                <group>
-                                    <field name="attachment_ids" widget="many2many_binary"/>
-                                </group>
-                                <group>
-                                    <field name="template_id" label="Use template" context="{'default_model': 'slide.channel.partner'}"/>
-                                </group>
-                            </group>
+                            <field name="attachment_ids" widget="mail_composer_attachment_list" invisible="not send_email"/>
                         </group>
                     </sheet>
                     <footer>
                         <button string="Send" invisible="not send_email" name="action_invite" type="object" class="btn-primary" data-hotkey="q"/>
                         <button string="Close" invisible="send_email" class="btn-secondary" special="cancel" data-hotkey="x"/>
                         <button string="Cancel" invisible="not send_email" class="btn-secondary" special="cancel" data-hotkey="x"/>
+                        <field name="attachment_ids" invisible="not send_email" widget="mail_composer_attachment_selector"/>
+                        <field name="template_id" invisible="not send_email" context="{'default_model': 'slide.channel.partner'}" widget="mail_composer_template_selector"/>
```

Comment:
Is the context still necessary? It believe that it was useful for the many2one widget. The `mail_composer_template_selector` seems to use the `render_model` field, which is "slide.channel.partner"